data/downloads/generate-timestamp-river-discharge-data.py

```diff
@@ -0,0 +1,101 @@
+import os
+import pandas as pd
+import geopandas as gpd
+from io import StringIO
+import json
+
+# Directory containing the CSV files
+data_dir = "/data/downloads/download/xco2"
```

Comment:
I think this needs to be changed

Reply:
To what?

Reply:
e.g. "/data/downloads/download/river_discharge"